build/run.js

```diff
@@ -70,7 +72,17 @@ function command(docs) {
     return {docs}
 }
 
-
+function run_project_manager() {
+    const bin_path = paths.get_project_manager_path(paths.dist.bin)
+    console.log('Starting language server from ' + bin_path)
+    child_process.execFile(bin_path, [], (error, stdout, stderr) => {
+        console.error(stderr)
+        if (error) {
+            throw error
```

Comment:
are we sure we want to break this thing on each error ? What if during development we will send bad message from rust to the project anager, we will see it in console, then we change rust to fix it - we dont want then the `watch` script to break, right?

Reply:
Depends. If the language server crashes, should we stop the watch script or try to restart the language server?

I think it might be better to also stop `watch` to have the log output from the crashing language server readily available. Otherwise, it will be gone by the time we realize what is going on from new status updates.

Reply:
If it executes when LS terinates, its ok. Please be sure it works this way after you make interactive terminal output here.